### PR TITLE
the bootctl changes from PR #9437 (the boot counting PR)

### PR DESCRIPTION
**Pull request by poettering**
As requested by @keszybz 

This also contains the commits from PR #10173, as they are required by this PR.

### CODE REVIEW

File: docs/ENVIRONMENT.md [hunk 1]
```diff
@@ -105,6 +105,13 @@ systemd-timedated:
   first existing unit listed in the environment variable, and
   `timedatectl set-ntp off` disables and stops all listed units.
 
+bootctl and other tools that access the EFI System Partition (ESP):
+
+* `$SYSTEMD_RELAX_ESP=1` — if set, the ESP validation checks are
```
Comment (keszybz) [inline docs/ENVIRONMENT.md]: Hmm, not crazy about the name. Maybe `SYSTEMD_ESP_CHECKS=0` instead? Or even `SYSTEMD_RELAX_ESP_CHECKS=1` ?

File: man/bootctl.xml [hunk 2]
```diff
@@ -115,7 +115,8 @@
 
         <listitem><para>Shows all available boot loader entries implementing the <ulink
         url="https://github.com/systemd/systemd/blob/master/docs/BOOT_LOADER_SPECIFICATION.md">Boot Loader
-        Specification</ulink></para></listitem>
+        Specification</ulink>, as well as all other and automatic entries the boot loader used on the current boot
```
Comment (keszybz) [inline man/bootctl.xml]: "all other and automatic" is grammatically suspicious and "discovered" (at the end) also seems misplaced. Maybe "as well as any other entries discovered or automatically generated by the boot loader" ?

**Comment by poettering** [comment]
> In one patch "Also fix a ">>" → ">" typo in the last line." seems misplaced: there's nothing in the patch about ">>".

Ah, this was fixed in the meantime by 08d541ca06c75c8539206c60eba975fb3ecd6460...

**Comment by poettering** [comment]
Force pushed a new version. No other changes than the three @keszybz suggested.

**Comment by keszybz** [comment]
LGTM.

**Comment by keszybz** [comment]
Same here, let's merge since the CI seems to have gone out to lunch. It passed before, so it should be OK.

**Pull request merged**